assets/src/edit-story/elements/text/display.js

```diff
@@ -100,6 +108,11 @@ function TextDisplay({
       : '';
   });
 
+  // @todo This logic is temporary and will change with selecting part + marking bold.
```

Comment:
To be clear, the goal is for this not be a flag on the element itself, but using the Draft.js API instead to wrap the text? Can't we do that already now? Otherwise we need to make sure not to forget a migration step for this later.

Reply:
Yep, it's doable via keyboard currently but not via the design panel. See also #413

Reply:
I'm actually not very certain this should be done via Draft.JS. The text markup, IMHO, should be our own. And we should convert Draft.JS output to our own markup if there's any delta.